README.md

```diff
@@ -68,9 +68,17 @@ T_w_i = sessions.trajectories[keys[0]]  # first pose, from sensor/rig to world
 
 :one: Install the core dependencies:
 
-- Python >= 3.9
-- [hloc v1.4](https://github.com/cvg/Hierarchical-Localization) and its dependencies, including [COLMAP 3.8](https://colmap.github.io/install.html) built from source.
-- [pyceres](https://github.com/cvg/pyceres) built from source at tag v1.0.
+- Python >= 3.9, we recommend to use the virtual environment of your choice.
```

Comment:
I ran into some troubles with Python > 3.10 + pycolmap 0.4.0. Maybe we can say here Python 3.9 / 3.10.

Similarly, miniconda created some headaches due to imcompatible system libs on Ubuntu 24 so maybe we shoul recommend python venv directly :)

Reply:
Ah, I see. I used venv.
Sure, I can change to more strict Python version.

Reply:
done.